tock/tock/templates/hours/reports_list.html

```diff
@@ -12,28 +12,33 @@ <h2>Regular Reports</h2>
 	<ul>
 		<li>Complete timecard data:
 			<a href="{% url 'reports:BulkTimecardList' %}">All</a>
-			<a href="{% url 'reports:BulkTimecardList' %}?after=2016-10-02">FY2017</a>
-            <a href="{% url 'reports:BulkTimecardList' %}?after=2017-10-01">FY2018</a>
+			{% for year in fiscal_years %}
+				<a href="{% url 'reports:BulkTimecardList' %}?after{{ year|add:"-1" }}-10-01&before={{ year }}-09-30">FY{{ year }}</a>
+			{% endfor %}
 		</li>
 		<li>Complete timecard data with fewer fields:
 			<a href="{% url 'reports:SlimBulkTimecardList' %}">All</a>
-			<a href="{% url 'reports:SlimBulkTimecardList' %}?after=2016-10-02">FY2017</a>
-            <a href="{% url 'reports:SlimBulkTimecardList' %}?after=2017-10-01">FY2018</a>
+			{% for year in fiscal_years %}
+				<a href="{% url 'reports:SlimBulkTimecardList' %}?after{{ year|add:"-1" }}-10-01&before={{ year }}-09-30">FY{{ year }}</a>
```

Comment:
I wonder if it'd be cleaner to add `prior_year` into context as well, in the view, instead of calcing it multiple places in the template.

Reply:
The logic has changed as well, so this is no longer relevant.